Scripts/Python/nxusBookMachine.py

```diff
@@ -1297,6 +1310,9 @@ def IUpdateCityLinksList(self):
             #special case: and another one for GuildPub name
             elif hasattr(ageData, 'guild'):
                 displayName = "The %s' Pub" % (ageData.guild)
+            #another special case for the new GoMe Pub
+            elif ageData.ageFilename == 'GoMePubNew':
+                displayName = "New Messengers Pub"
```

Comment:
I think we need to remove this in favor of... well, something not hardcoded.

Reply:
Sure! If you have another suggestion, I'll be happy to change it.

Reply:
Can we add an entry to a loc file and use `PtGetLocalizedString()`?

Reply:
> Can we add an entry to a loc file and use `PtGetLocalizedString()`?

Done (I think). Also added the message to the Nexus LOCs in the associated PR on moul-assets.